emmaa/model.py

```diff
@@ -922,3 +927,120 @@ def load_custom_grounding_map(model, bucket=EMMAA_BUCKET_NAME):
 def get_search_term_names(model, bucket=EMMAA_BUCKET_NAME):
     config = load_config_from_s3(model, bucket=bucket)
     return [st['name'] for st in config['search_terms']]
+
+
+def pysb_to_gromet(pysb_model, model_name, fname):
```

Comment:
Could you include this in the INDRA PySB assembler?

Reply:
We discussed this and decided to keep in EMMAA since it will be only used for EMMAA